src/transformers/utils/quantization_config.py

```diff
@@ -1090,6 +1090,8 @@ class CompressedTensorsConfig(QuantizationConfigMixin):
             configuration for sparsity compression
         quant_method (`str`, *optional*, defaults to `"compressed-tensors"`):
             do not override, should be compressed-tensors
+        run_compressed (`bool`, *optional*, defaults to `True`): compress weights (usually linear) modules
+            as compressed weights, following the method specified on `format` arg above
```

Comment:
```suggestion
        run_compressed (`bool`, *optional*, defaults to `True`): alter submodules in order to emulate compressed model execution if True, otherwise decompress modules
```

Reply:
ok the doc strings is a bit confusing - ill fix